demo/.gitignore

```diff
@@ -0,0 +1,2 @@
+/postgres/data/*
```

Comment:
it may be worth parking this under one level in the hierarchy if there are other demos not of the same time... 

e.g. demo/validate/ demo/postgres-system (I don't have great names for them)
This can always wait for the future.

Reply:
I like the thought.  I had been thinking about maybe just having alternative docker-compose files, but maybe different directories will be better...

Lets use what I have for now and I will chew on this going forward.